includes/virtual-machines-image-builder-overview.md

```diff
@@ -46,6 +46,12 @@ AIB will support Azure Marketplace base OS images:
 - Windows 2016
 - Windows 2019
 
+AIB will support RHEL ISO's, as a source for:
+- RHEL 7.3
+- RHEL 7.4
+- RHEL 7.5
+
+RHEL 7.6 is not supported, but being tested.
 
```

Comment:
@danielsollondon @cynthn - not sure if this is confusing to anyone else, but the section of the article seems to say that RHEL 7.6 is supported and is not supported. See:

![image](https://user-images.githubusercontent.com/43421140/60464082-32141780-9c13-11e9-9fdb-055881ce31a3.png)

if this is as intended, let me know and I'll go ahead and merge... just thought I should check. Thanks.